src/main/java/net/macmv/rgen/block/RgenTrapDoors.java

```diff
@@ -0,0 +1,16 @@
+package net.macmv.rgen.block;
+
+import net.minecraft.block.BlockTrapDoor;
+import net.minecraft.block.SoundType;
+import net.minecraft.block.material.Material;
+
+public class RgenTrapDoors extends BlockTrapDoor {
+
+  public RgenTrapDoors() {
+    super(Material.WOOD);
+    this.setHardness(3.0F);
+    this.setResistance(5.0F);
+    this.setSoundType(SoundType.WOOD);
+  }
+
+}
```

Comment:
this organization is great, do this more.

Reply:
485b8d30ab4a547b61caefdb359946de5f08bb33 resolved